content/techniques/history.md

```diff
@@ -90,4 +90,4 @@ MoveInvariant ==
 
 ### Limitations
 
-Multistep invariants get clunky when you have multiple processes, since you have to store the state, action, and process. Additionally, it gets less useful when applying invariants to complex states.
\ No newline at end of file
+Multistep invariants get clunky when you have multiple processes, since you have to store the state, action, and process. Additionally, it gets less useful when applying invariants to complex states.
```

Comment:
Hmm, that's weird, I didn't make a change there! Probably some automatic line-ending thing from Github's online editor.